### ISSUE TITLE
Forget Password 

### ISSUE THREAD

**Issue by Vedsaga** [Monofyi/Mobile#28]
Steps to rest the password. 

1.Check Mail 
![Screenshot_20210723_184556](https://user-images.githubusercontent.com/59083528/126787166-50b34ed8-a7dc-4e18-9986-f2640ba685ef.jpg)

2. Filling OTP
![Screenshot_20210723_184552](https://user-images.githubusercontent.com/59083528/126787956-f2754fab-8c3d-4917-acd5-f5123f664772.jpg)

3. Once OTP is validated filling the new password 
![Screenshot_20210723_184420](https://user-images.githubusercontent.com/59083528/126788222-447c9997-19ad-4028-8d4a-8854d24cd526.jpg)

4. Confirmation screen
![Screenshot_20210723_185256](https://user-images.githubusercontent.com/59083528/126788106-2a3b2580-754e-4c63-824b-d10b151ac3e8.jpg)

**Comment by Vedsaga** [comment]
Sorry, it's my mistake I forgot to ask... current from where we using thoes illustrate for the app? Are thoes files are in `.jpg` format or `.svg` format?

**Comment by arun-muralidharan** [comment]
I export them as `.svg` from Figma, unless that doesn't work in Flutter. In that case, I export them as `.png` and then convert them to `.svg`. So, all images within the app are `.svg`.

**Comment by Vedsaga** [comment]
Ohh, actually i should have informed in the beginning most illustration looks blur... Should, I export all the illustrations used during deisgin and share on discord? Does Flutter support `.svg` rending?

**Comment by arun-muralidharan** [comment]
We are using the `flutter_svg` package to display the `.svg` images. However, it does not support `.svg`s that use the pattern feature. That is the reason I had to convert some of the images to `.png` and reconvert them into `.svg`. The blurriness is likely due to the resolution loss between these conversions.

Changing the images is not too difficult. If you can share the `.svg` files straight from the source (without using the pattern feature), we can replace them and it won't be blurry.

**Comment by Vedsaga** [comment]
I can group all the `.svg` and make it as single entity.. Finger cross hope it then render... Please allow me 5-10 min I will share immediately

**Comment by Vedsaga** [comment]
Anna, are we working on Forget Password currently?

**Comment by arun-muralidharan** [comment]
Yes, I asked backend and the API is ready. I'll be working on it today.

**Comment by arun-muralidharan** [comment]
@Vedsaga Sending OTP to reset password requires the user to type in their username and email (answering recovery question looks to be optional). So, we may need one more page in this module before the OTP screen, where the user has to give us their username and email.

@rexdivakar There may be cases where the user remembers only their username or only their email. Is it possible to send an OTP even if they fill only one of these two fields? Currently, the API requires both.

**Comment by Vedsaga** [comment]
I thought we are going to skip the forget password and will work on it ending phase. That why I marked it as backlog. Anyway worries, Surely I csn design the screen in just few minutes...

**Comment by Vedsaga** [comment]
Yaa Anna sure, it's not important right now.. As I was also having confusion about what fields should be thier so, after discussing with Surya Anna and Divakar Anna we decided to skip it... We can move on to `Supplier List`
